readme.md

```diff
@@ -75,13 +75,13 @@ flowchart TD
 - [x] Released VS-Code extension with basic auto-complete from curated modules in the farm-repo.
 - [x] Documented Terrarium Platform Framework and App dependencies format with example.
 - [x] Implemented CLI Command to lint & parse Terrarium Platform Templates.
-- [ ] Removing complexity of installing Postgres & T8-API in Docker to run VS Code extension, by abstracting it into CLI.
-- [ ] Developing CLI Command to compose working Terraform templates using T8-Platform Templates & App Dependencies.
-- [ ] Improving VS Code Plugin to support auto-complete with local modules.
-- [ ] Developing UI to assist developers in selecting App Dependencies by rendering platform and farm insights.
-- [ ] Adding taxonomy & dependency-interfaces content to the farm-repo.
```

Comment:
Why is this being removed? Are we not doing this?

Reply:
The dependency interface was done, so I had to split it out, and I wasn't sure if taxonomy mapping should seen as an implementation detail or expose to the user here in OS progress.
I've added it back for now to avoid this confusion.